src/devices/Gpio/Drivers/VirtualGpioPin.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.Gpio;
+
+namespace Iot.Device.Gpio.Drivers
```

Comment:
It's only internal, but this looks like the wrong namespace.

Reply:
great catch, removed the "driver" part